packages/api/src/sequelize/migrations/2025-04-16_00_remove_ehr_contraints.ts

```diff
@@ -0,0 +1,56 @@
+import type { Migration } from "..";
+
+const cxMappingTableName = "cx_mapping";
+const cxMappingTableIndexName = "cx_mapping_source_externalId_index";
```

Comment:
IIRC, PG it's case insensitive, so prob can just write those in lowercase, like `cx_mapping_source_externalid_index`